### PR TITLE
Additional CSS changes to support styling the casino page

### PR DESCRIPTION
**Pull request by tmathern**
Fix multiple small CSS issues.

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/casino
- After: https://casino-css-fixes--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/casino

## 📝 Description:
Additional CSS changes to support styling the casino page:
- adds classes/metadata for text(title and content) alignment and color
- fixes stats to be centered no matter how many columns there are

### CODE REVIEW

**Comment by tmathern** [comment]
Also:  https://casino-css-fixes--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/shows

**Comment by tmathern** [comment]
https://casino-css-fixes--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/hotel

File: caesars-palace/blocks/carousel/carousel.css [hunk 1]
```diff
@@ -79,10 +79,10 @@
 }
 
 .carousel.block .carousel-text h1 {
-  font-size: 32px;
   text-transform: uppercase;
   line-height: 1.1;
   font-weight: var(--emphasized-font-weight);
+  font-size: var(--heading-font-size-xl);
```
Comment (tmathern) [inline caesars-palace/blocks/carousel/carousel.css:85]: Fixes wrong font size

File: caesars-palace/blocks/stats/stats.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default function decorate(block) {
```
Comment (tmathern) [inline caesars-palace/blocks/stats/stats.js:1]: Adapt grid layout depending on how many items to show

File: caesars-palace/styles/styles.css [hunk 3]
```diff
@@ -53,7 +53,7 @@
   --heading-font-size-hero: 64px;
   --heading-font-size-xxl: 54px;
   --heading-font-size-xl: 36px;
-  --heading-font-size-teaser: 20px;
+  --heading-font-size-teaser: 26px;
```
Comment (tmathern) [inline caesars-palace/styles/styles.css:56]: Fixes wrong font size

File: caesars-palace/styles/styles.css [hunk 4]
```diff
@@ -295,6 +295,27 @@ main .section {
   padding: 0 var(--layout-size-m);
 }
 
+main .section.has-centered-text > div > p {
```
Comment (tmathern) [inline caesars-palace/styles/styles.css:298]: Adds some smaller CSS classes to use in metadata section to control text-alignment and text-color

File: caesars-palace/styles/styles.css [hunk 5]
```diff
@@ -304,6 +325,10 @@ main .section.has-background {
   padding: 0;
 }
 
+main .section.has-background-overlay {
```
Comment (tmathern) [inline caesars-palace/styles/styles.css:328]: Adds a class to define when a background (image) should have a colored overlay

**Comment by tmathern** [comment]
https://casino-css-fixes--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/deals

**Comment by tmathern** [comment]
https://casino-css-fixes--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/race-and-sports-book

**Comment by tmathern** [comment]
https://casino-css-fixes--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/wifi

**Review by amol-anand — APPROVED** [review]

**Pull request merged**